docs/tutorials/contributing/strategy/writing_test_for_the_new_strategy.rst

```diff
@@ -65,66 +92,60 @@ All three of these functions can take an optional keyword argument
 
    This is equivalent to::
 
-    def test_effect_of_strategy(self):
-        P1 = axelrod.TitForTat() # Or whatever player is in your test class
-        P2 = axelrod.Player()
-        P2.history = []
-        P2.history = []
-        self.assertEqual(P1.strategy(P2), 'C')
+    P1 = axelrod.TitForTat() # Or whatever player is in your test class
+    P2 = axelrod.Player()
+    self.assertEqual(P1.strategy(P2), 'C')
 
-2. The member function :code:`markov_test` takes a list of four plays, each
+2. The member function :code:`second_play_test` takes a list of four plays, each
    following one round of CC, CD, DC, and DD respectively::
 
-    def test_effect_of_strategy(self):
-        self.markov_test(['C', 'D', 'D', 'C'])
+    self.second_play_test('C', 'D', 'D', 'C')
 
-   This is equivalent to::
-
-    def test_effect_of_strategy(self):
-        P1 = axelrod.TitForTat() # Or whatever player is in your test class
-        P2 = axelrod.Player()
-        P2.history = ['C']
-        P2.history = ['C']
-        self.assertEqual(P1.strategy(P2), 'C')
-        P2.history = ['C']
-        P2.history = ['D']
-        self.assertEqual(P1.strategy(P2), 'D')
-        P2.history = ['D']
-        P2.history = ['C']
-        self.assertEqual(P1.strategy(P2), 'D')
-        P2.history = ['D']
-        P2.history = ['D']
-        self.assertEqual(P1.strategy(P2), 'C')
+   This is equivalent to choosing an opponent will play C or D as needed and
```

Comment:
I'm happy to rework the doc -- just note that we're reviewing the existing docs at this point.